book/chapter-01.md

```diff
@@ -1,133 +1,139 @@
 # Chapter 1: Centralized Payment Processor
 
-// TODO
+**Introduction**
+In Chapter 1 we will learn about cryptographic primitives and how they have been used to build Bitcoin and Ethereum. We will also explore the design of an centralised computer system; our Web 2.0 Paypal payment processor and and also discuss the pros/cons of centralized systems.
```

Comment:
Removed a few typos: "In Chapter 1 we will learn about cryptographic primitives and how they have been used to build Bitcoin and Ethereum. We will also explore the design of a centralised computer system; our Web 2.0 PayPal payment processor and also discuss the pros/cons of centralized systems."